models/stg_recharge__payment_source.sql

```diff
@@ -33,15 +33,15 @@ final as (
         card_exp_year,
         card_last_4,
         has_card_error_in_dunning,
-        billing_address_first_name || ' ' || billing_address_last_name as billing_address_full_name,
-        billing_address_address_1 as billing_address_line_1,
-        billing_address_address_2 as billing_address_line_2,
-        billing_address_city as billing_address_city,
-        billing_address_province,
-        billing_address_zip,
-        billing_address_country,
-        billing_address_company,
-        billing_address_phone,
+        billing_address_first_name || ' ' || billing_address_last_name as billing_address_full_name, 
+        billing_address_address_1 as billing_address_line_1, 
+        billing_address_address_2 as billing_address_line_2, 
+        billing_address_city as billing_address_city, 
+        billing_address_province, 
+        billing_address_zip, 
+        billing_address_country, 
+        billing_address_company, 
+        billing_address_phone, 
```

Comment:
Same comment about these all being custom fields. We should let the variables handle these.